src/pages/map.tsx

```diff
@@ -25,3 +26,27 @@ export default function MapPage() {
     </Layout>
   );
 }
+
+export async function getStaticProps() {
+  const enabled =  process.env.CANOPY_CONFIG.map.enabled;
+
+  if (!enabled) {
+    return {
+      notFound: true,
```

Comment:
I don't know the answer, just curious if this should be returning a `404` via the `notFound` property?   @mathewjordan ?

Reply:
`notFound: true` will 404 the route and output the 404 page content. The page will still technically be generated by Next but it is not a 200. I think this is what we want. This is same method we use in Northwestern's DC project.